pkg/sdn/plugin/poddb.go

```diff
@@ -0,0 +1,46 @@
+package plugin
```

Comment:
@danwinship there's no good way to pull the runtime out of kubelet since openshift doens't even get the kubelet object. The best we can do is continue to provide our own kubelet network plugin, but as a proxy for the actual kubelet CNI driver, and then save the Host in our Init() call before passing the init to CNI.

So we take a step backwards and can no longer use the normal kubelet CNI plugin, but I guess it works for now.  See how you like the changes I pushed, especially pkg/sdn/plugin/plugin.go.